### DISCUSSION TITLE
Git instructions for everyone

### DISCUSSION THREAD

**Discussion by anaricamaemendoza** [MMS-142-Group-K-2024/mms142-groupk-2024#3] [Announcements]
Hello, everyone! Thank you for participating in our poll last night. Here are the instructions to follow. If you have any questions, feel free to reach out for clarification. You can also ask our classmates for help in case I’m unavailable due to my work schedule.

Google docs for a bit more familiar navigation: https://docs.google.com/document/d/1g-kazFXj4mpbI_AIe1bBXC8dGXSQgbYxLKodWrS8vZA/edit?usp=sharing

Git pages for easy access: https://github.com/MMS-142-Group-K-2024/mms142-groupk-2024/wiki/Detailed-instructions-for-all-(after-establishing-subpages)

Feel free to comment below if you can see this or have your questions. Thanks!

**Comment by AyessaTM** [comment]
Got it! Thanks for the instructions ate!

**Comment by miryam37** [comment]
Hello po. Thank you po sa instructions.

**Comment by DianneLouiseNituma** [comment]
Got it! Thank you sa instructions!

**Comment by pennyxx28** [comment]
Hi! Got it! Thank you for the instructions!
